CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to
 ### Added
 
 - Emit error if contract macro is above entry_points
-- Remove support for `, custom(msg=.. ,query=..)` in entry_points
+- Deprecate support for `, custom(msg=.. ,query=..)` in entry_points
```

Comment:
It is remove in 1.2.0, and then "reverted and marked as deprecated in 1.2.1". Don't try to hide stuff that happened.